### PR TITLE
Unsupported upgrade configs

### PR DESCRIPTION
**Pull request by dmpop**
Add info about unsupported configurations as per https://jira.suse.com/browse/SOC-10496

### CODE REVIEW

File: xml/soc-operations-maintenance.xml [hunk 1]
```diff
@@ -482,6 +482,107 @@
     </listitem>
    </itemizedlist>
   </section>
+
+  <section xml:id="sec-upgrade-unsupported-configs">
+   <title>Unsupported configurations</title>
+   <para>
+    In &cloud; 9, certain configurations and barclamp combinations are no
+    longer supported. See the &cloud; 9 release notes for details. The upgrade
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: Can we link to the release notes?

File: xml/soc-operations-maintenance.xml [hunk 2]
```diff
@@ -482,6 +482,107 @@
...
+    <listitem>
+     <para>
+       As of &cloud; 9, the Xen hypervisor is no longer supported. On any
+       cloud, where Xen based compute nodes are still present, the following
+       procedure must be followed before it can be upgraded to &cloud; 9:
+     </para>
+     <itemizedlist>
+    <listitem>
+     <para>
+      Run <command>nova-service disabled</command> on all &o_comp; &compnode;s
```
Comment (jgrassler) [inline xml/soc-operations-maintenance.xml]: Yikes. I really messed up that part of the document (I checked, the errors are present in my asciidoc text already) now that I look at it again in a different format. There are two things wrong with this:

1. `nova service-disable` (without the `d` at the end and with the dash moved)
2. Run `nova service-disable` _for_ all compute nodes - that command is being run on a controller for every compute node to be disabled.

Comment (dmpop) [reply xml/soc-operations-maintenance.xml]: No problem whatsoever! Fixed. :-)

File: xml/soc-operations-maintenance.xml [hunk 3]
```diff
@@ -482,6 +482,109 @@
...
    </itemizedlist>
   </section>
+
+  <section xml:id="sec-upgrade-unsupported-configs">
+   <title>Unsupported configurations</title>
+   <para>
+    In &cloud; 9, certain configurations and barclamp combinations are no
+    longer supported. See the <link
+    xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-OPENSTACK-CLOUD/9/">&cloud;
+    9 release notes</link> for details. The upgrade prechecks fail if they
```
Comment (jsuchome) [inline xml/soc-operations-maintenance.xml]: The pdf shows the URL as visible part of the text, is that correct?

Comment (dmpop) [reply xml/soc-operations-maintenance.xml]: Yes, PDF show the full link (useful when you print docs).

**Pull request merged**